spec/api/v1/api_crops_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe Api::V1::CropsController do
+
+  before(:each) do
+    @crop = FactoryGirl.create(:tomato)
+    @crop.save
+  end
+
+  describe "GET crops index" do
+    it 'fetches the crops index' do
+      get '/api/v1/crops'
+      response.should be_success
+      response.content_type.should eq("application/json")
+    end
```

Comment:
i'd like to see some tests here for the actual content of the JSON, i.e. that it contains a list with one item, which is tomato.